pandas/tests/tools/test_to_numeric.py

```diff
@@ -725,3 +725,19 @@ def test_to_numeric_from_nullable_string(values, expected):
     s = Series(values, dtype="string")
     result = to_numeric(s)
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "data, input_dtype, downcast, expected_dtype",
+    (
+        ([1, 1], "Int64", "integer", "Int8"),
+        ([450, 300], "Int64", "integer", "Int16"),
+        ([1, 1], "Float64", "float", "Float32"),
+        ([1, 1], "Float64", "integer", "Int8"),
```

Comment:
Can you add some cases with missing values?

Reply:
Done

Reply:
Thanks!

Does it also work when starting with Float data *with* NA to downcast to integer?

Reply:
Yes. Added those cases to the test in 23c4ae6